simple_ado/workitems.py

```diff
@@ -529,15 +534,16 @@ def delete(
         request_url += "&api-version=4.1"
 
         response = self.http_client.delete(
-            request_url, additional_headers={"Content-Type": "application/json-patch+json"}
+            request_url,
+            additional_headers={"Content-Type": "application/json-patch+json"},
         )
 
         if response.status_code != 204:
             raise ADOHTTPException(f"Failed to delete '{identifier}'", response)
 
         return self.http_client.decode_response(response)
 
-    def batch(self, operations: List[BatchRequest]) -> ADOResponse:
+    def batch(self, operations: typing.List[BatchRequest]) -> ADOResponse:
```

Comment:
Shouldn't we use a lowercase `list` here? Same goes for other places as well.

Reply:
Ideally, yes. However, in this class there is a method called `list` so if you try and set the annotation, it attempts to call that method instead.